### PR TITLE
refactor: componentization of reset button

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

image

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/store/seriesData.ts [hunk 1]
```diff
@@ -138,16 +139,20 @@ const seriesData: StoreModule = {
...
 
       this.notify(state, 'zoomRange');
+      this.notify(state, 'isZooming');
     },
     resetZoom({ state, initStoreState }) {
       const { series, options } = initStoreState;
       const { rawCategories } = state;
 
       state.zoomRange = initZoomRange(series, options, rawCategories);
+      state.isZooming = false;
```
Comment (shiren) [inline src/store/seriesData.ts:152]: isZooming은 스토어에 게터로 구현하면 효율적일 것 같아요.
게터는 그동안 잘 안써서 걱정이지만 문제가 있으면 해결해서라도 게터를 사용하면 어덜까요?

File: src/store/seriesData.ts [hunk 2]
```diff
@@ -138,16 +139,20 @@ const seriesData: StoreModule = {
...
 
       this.notify(state, 'zoomRange');
+      this.notify(state, 'isZooming');
```
Comment (shiren) [inline src/store/seriesData.ts:155]: 얘는 그냥 값이니까 값을 셋팅하면 되는거아니에요? notify할 필요가 없을것 같은데

**Review by shiren — COMMENTED** [review]
수고하셨습니다.

**Review by lja1018 — APPROVED** [review]
[09/16] 리뷰완료
고생하셨습니다!

**Review by jungeun-cho — APPROVED** [review]
[9/16] 이견없습니다.

**Pull request merged**